databricks/koalas/indexes.py

```diff
@@ -2444,7 +2455,7 @@ def from_frame(df, names=None):
 
         index_map = OrderedDict(zip(sdf.columns, names))
         internal = InternalFrame(spark_frame=sdf, index_map=index_map)
-        return DataFrame(internal).index
+        return DataFrame(internal).index  # type: ignore
```

Comment:
Otherwise, Mypy expects "Index" as the return type.

Reply:
Oh, so maybe it seems that `mypy` requires us to add a kind of abstract methods to the `Index` class ?

Maybe can we ignore that rule like we did for `inplace` parameter ??  😲

Reply:
Thanks @itholic, this is a good idea!

Unfortunately, I didn't find a parameter in https://mypy.readthedocs.io/en/stable/config_file.html can ignore this rule.

I do agree this should a common problem for other projects. Let me do more search and see if there is a better solution.

Reply:
Yep, Sure :)